service-front/module/Application/config/module.config.php

```diff
@@ -110,6 +110,7 @@
             'formRadio'             => 'Application\Form\View\Helper\FormRadio',
             'formCheckbox'          => 'Application\Form\View\Helper\FormMultiCheckbox',
             'finalCheckAccessible'  => 'Application\View\Helper\FinalCheckAccessible',
+            'formtext'              => 'Application\Form\View\Helper\FormText',
```

Comment:
It would be good to understand a little bit more about how this overrides the default formtext helper. Could you perhaps add a comment or some notes somewhere which explain why we have done this here? This would help give some context to anyone coming along later (e.g. if Laminas ever fix the issue themselves, we'd want to know where in our code we would need to come back to and change).

Reply:
I've left a comment in the config file, linking to the issue. But after looking at the documentation I don't think it's best to start putting base Laminas documentation links into the codebase.

The links that helped me though -
https://docs.laminas.dev/laminas-view/helpers/intro/
https://docs.laminas.dev/laminas-form/helper/abstract-helper/ 

I've dropped them in the PR as well.